manifest.json

```diff
@@ -1,17 +1,15 @@
 {
   "name": "admin-iframe-container",
   "vendor": "vtex",
-  "version": "1.1.0",
+  "version": "1.2.0-beta.4",
```

Comment:
Will you keep this beta version here?

Reply:
I'll release a 2.x beta version. Once is all good,  2.x stable.